package.json

```diff
@@ -60,6 +60,11 @@
 					"default": true,
 					"description": "Enable Lean input mode"
 				},
+				"lean.input.markdown" : {
```

Comment:
I'd prefer a `lean.input.languages` configuration option that takes a list as argument, defaulting to `["lean"]`.

Reply:
done